lib/Onyx.ts

```diff
@@ -616,23 +599,42 @@ function update(data: OnyxUpdate[]): Promise<void> {
         }
     });
 
+    const updateQueue: Record<OnyxKey, Array<OnyxValue<OnyxKey>>> = {};
```

Comment:
Please add some comments about what this queue is for, what it's format is, and how it should be used.